### PR TITLE
feat: Add testimonial carousel

### LINKED ISSUE #25: [Feature] Testimonial components
### Description

Testimonial component for the website is required 

### Screenshots

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by prnvtripathi**
## Fixes Issue
 closes #25 
## Changes proposed

- made a testimonial carousel

### CODE REVIEW

**Comment by prnvtripathi** [comment]
Working on it 😁

**Comment by prnvtripathi** [comment]
hey @AbhiPatel10 , please check the new changes made and do let me know if any suggestions are there :)

File: src/pages/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import Head from "next/head";
 import Script from "next/script";
 import Home from "./home";
+import Navbar from "@/components/Navbar";
```
Comment (AbhiPatel10) [inline src/pages/index.js:4]: Remove this line

File: src/components/testimonial.js
Comment (AbhiPatel10) [inline src/components/testimonial.js:1]: Add this component in home page and also add header component with testimonial section Name

**Review by AbhiPatel10 — CHANGES_REQUESTED** [review]

File: src/components/testimonial.js
Comment (prnvtripathi) [reply src/components/testimonial.js:1]: I added the component in the home page, where do I have to add header component?

Comment (AbhiPatel10) [reply src/components/testimonial.js:1]: In Your component.

**Comment by himanshu1221** [comment]
@prnvtripathi any update on this ?

**Comment by prnvtripathi** [comment]
I've done all the updates, except the header thing, i am not able to understand what to do in it.

**Comment by himanshu1221** [comment]
Share any screenshots if possible

**Review by AbhiPatel10 — APPROVED** [review]

**Review by prnvtripathi — COMMENTED** [review]
@himanshu1221 please approve the changes, as it is required to merge the changes

**Review by himanshu1221 — APPROVED** [review]
LGTM:+1:

**Pull request merged**